src/widgets/List.tsx

```diff
@@ -46,6 +46,7 @@ interface ListItemPropsType {
   insetChildren?: boolean;
   nestedItems?: React.ReactNode[];
   selected?: boolean;
+  ListItemTextProps?: ListItemTextProps;
```

Comment:
This is a very very leaky abstraction (not really an abstraction)...

Just call the prop `secondaryTextStyle` and pass just the style itself or something. Or if it's not a style, then just call it, `secondaryTextColor`.